ecf/mesoscale/stats/jevs_rap_grid2obs_stats.ecf

```diff
@@ -1 +1,110 @@
-sample
+#!/bin/bash
+#PBS -N jevs_rap_grid2obs_stats_00
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q "dev"
+#PBS -A VERF-DEV
+#PBS -l walltime=11:59:00
+#PBS -l select=1:ncpus=1:mem=12GB
+#PBS -l debug=true
+
+set -x
+  export model=evs
+  export machine=WCOSS2
+
+# ECF Settings
+# export RUN_ENVIR=nco
+  export RUN_ENVIR=emc
+  export SENDECF=YES
+  export SENDCOM=YES
+  export KEEPDATA=YES
+  export SENDDBN=YES
+  export SENDDBN_NTC=
+  export job=${PBS_JOBNAME:-jevs_mesoscale_grid2obs_stats}
+  export jobid=$job.${PBS_JOBID:-$$}
+  export SITE=$(cat /etc/cluster_name)
+  export USE_CFP=YES
+  export nproc=128
+
+# General Verification Settings
+  export NET="evs"
+  export STEP="stats"
+  export COMPONENT="mesoscale"
+  export RUN="atmos"
+  export VERIF_CASE="grid2obs"
+  export MODELNAME="rap" 
+
+# export envir="dev"
+  export envir="prod"
+  export evs_run_mode="standalone"
+
+# EVS Settings
+  export HOMEevs="/lfs/h2/emc/vpppg/noscrub/${USER}/verification/EVS"
+# export HOMEevs=/lfs/h2/emc/vpppg/noscrub/${USER}/EVS
+# Subdirectories to EVS Home Directory
+  export PARMevs=$HOMEevs/parm
+  export USHevs=$HOMEevs/ush
+  export EXECevs=$HOMEevs/exec
+  export FIXevs="/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix"
+  export SCRIPTSevs=$HOMEevs/scripts
+  export MET_PLUS_CONF="${PARMevs}/metplus_config/mesoscale/grid2obs/stats"
+
+# EVS configuration
+  export config=$HOMEevs/parm/evs_config/mesoscale/config.evs.prod.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.${MODELNAME}
+
+# Load Modules
+
+source $HOMEevs/versions/run.ver
+
+source /usr/share/lmod/lmod/init/sh
+module reset
+export HPC_OPT=/apps/ops/para/libs
+export MET_bin_exec="bin"
+
+module load cray-pals/${craypals_ver}
+module load PrgEnv-intel/${PrgEnv_intel_ver}
+module load craype/${craype_ver}
+module load intel/${intel_ver}
+module load cray-mpich/${craympich_ver}
+module load cray-pals/${craypals_ver}
+module load cfp/${cfp_ver}
+module load libjpeg/${libjpeg_ver}
+module load libpng/${libpng_ver}
+module load zlib/${zlib_ver}
+module load jasper/${jasper_ver}
+module load udunits/${udunits_ver}
+module load gsl/${gsl_ver}
+module load hdf5/${hdf5_ver}
+module load python/${python_ver}
+module load netcdf/${netcdf_ver}
+module load nco/${nco_ver}
+module load prod_util/${prod_util_ver}
+module load prod_envir/${prod_envir_ver}
+module load cdo/${cdo_ver}
+module use /apps/ops/para/libs/modulefiles/compiler/intel/${intel_ver}
+module load grib_util/${grib_util_ver}
+module load wgrib2/${wgrib2_ver}
+module load met/${met_ver}
+module load metplus/${metplus_ver}
+
+export MET_PLUS_PATH="/apps/ops/para/libs/intel/19.1.3.304/metplus/${metplus_ver}"
+export MET_PATH="/apps/ops/para/libs/intel/19.1.3.304/met/${met_ver}"
```

Comment:
update MET_PATH to use ${intel_ver} instead of 19.1.3.304. Hard coded version numbers can only be in `versions/run.ver`.